pipeline/metadata/4.3.yaml

```diff
@@ -0,0 +1,534 @@
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
+- name: "test-rgw-ssl-image"
+  suite: "suites/nautilus/rgw/tier-1_rgw_ssl_containerized-regression.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
+- name: "test-upgrade-of-rpm-based-deployments"
+  suite: "suites/nautilus/upgrades/tier-1_upgrade_rpm.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-1_upgrade.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-3
+- name: "test-rgw-single-site-upgrade"
+  suite: "suites/nautilus/rgw/tier-1_rgw_test-4x-upgrade-to-latest.yaml"
+  global-conf: "conf/nautilus/rgw/tier-0_rgw.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
+- name: "test-rgw-multi-site-secondary-to-primary"
+  suite: "suites/nautilus/rgw/tier-1_rgw_multisite-secondary-to-primary.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw_multisite.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
+- name: "test-rgw-ecpool-multisite-primary-to-secondary"
+  suite: "suites/nautilus/rgw/tier-1_rgw_ecpool_verifying-data-from-primary.yaml"
+  global-conf: "conf/nautilus/rgw/ms-ec-profile-4+2-cluster.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
+- name: "test-rhceph-rhel-7-rpm-deployment-psi-only"
+  suite: "suites/nautilus/ansible/tier-1_deploy.yaml"
+  global-conf: "conf/nautilus/ansible/tier-1_deploy.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - ansible
```

Comment:
Ansible is not the FG name. You can remove it and add dmfg at all the ansible test suites.

Reply:
removed Ansible and added dmfg as FG name